moddingway/commands/note_commands.py

```diff
@@ -3,6 +3,36 @@
 from moddingway.util import is_user_moderator
 from .helper import create_logging_embed, create_response_context
 from moddingway.services import note_service
+from moddingway.database import notes_database
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class NoteDeleteView(discord.ui.View):
+    def __init__(self, note_id: int, interaction: discord.Interaction, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.note_id = note_id
+        self.original_interaction = interaction
+
+    @discord.ui.button(label="Confirm", style=discord.ButtonStyle.success)
+    async def first_button_callback(
```

Comment:
using `first` and `second` isnt really useful in this context, we can call these `confirm_button_callback` and `cancel_button_callback` instead